delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/ProjectsRoutes.scala

```diff
@@ -149,18 +144,10 @@ final class ProjectsRoutes(
                     },
                     // Deprecate/delete project
                     (delete & pathEndOrSingleSlash) {
-                      parameters("rev".as[Long], "prune".?(false)) {
```

Comment:
Removing more delete stuff (as for example the uuid is computed from the datastax driver library)